app/src/test/java/ingsw/group1/findmyphone/RandomGeoPositionGenerator.java

```diff
@@ -0,0 +1,27 @@
+package ingsw.group1.findmyphone;
+
+import java.util.Random;
+
+import ingsw.group1.findmyphone.location.GeoPosition;
+
+/**
+ * Class made to generate pseudo-random GeoPositions.
+ *
+ * @author Riccardo De Zen.
+ */
+public class RandomGeoPositionGenerator {
+
+    private static Random random = new Random();
+
+    /**
+     * Returns a randomly generated GeoPosition.
+     *
+     * @return a GeoPosition, with valid latitude and longitude.
+     */
+    public GeoPosition getRandomPosition() {
+        double latitude, longitude;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5010214)